api/apps/api/src/modules/geo-features/geo-feature.api.entity.ts

```diff
@@ -18,7 +19,7 @@ export interface GeoFeatureProperty {
 }
 
 @Entity('features')
-export class GeoFeature {
+export class GeoFeature extends BaseEntity {
```

Comment:
had to make this to be able to access the `id` of the entity after inserting it into the DB